.github/CODEOWNERS

```diff
@@ -1886,3 +1886,12 @@ spec/uploaders/supporting_evidence_attachment_uploader_spec.rb @department-of-ve
 spec/uploaders/uploader_virus_scan_spec.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/uploaders/validate_pdf_spec.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/prescription_details.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+modules/my_health/spec/request/v1/prescriptions_request_spec.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas/prescriptions_list_paginated.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas/prescriptions_list.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas/prescription_list_filtered.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas_camelized/prescription_list_filtered.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas_camelized/prescriptions_list_paginated.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas_camelized/prescriptions_list.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
```

Comment:
There is a section in this file (starting at around line 1620) where `spec/support/schemas` and `spec/support/schemas_camelized` are listed. Can you add these to that section?